pkg/block/metrics.go

```diff
@@ -134,3 +137,59 @@ func (m *MetricsAdapter) GetRegion(ctx context.Context, storageID, storageNamesp
 func (m *MetricsAdapter) RuntimeStats() map[string]string {
 	return m.adapter.RuntimeStats()
 }
+
+type Histograms struct {
```

Comment:
Consider calling this a Histogram - after all that's literally the name Prometheus gives it.